src/query/datavalues/Cargo.toml

```diff
@@ -14,6 +14,7 @@ test = false
 # Workspace dependencies
 common-arrow = { path = "../../common/arrow" }
 common-exception = { path = "../../common/exception" }
+common-hashtable = { path = "../../common/hashtable" }
```

Comment:
by the way. maybe we should move `hastable` crate into `base`

Reply:
let's do it after we have saha.